docs-js/orchestration/chat-completion.mdx

```diff
@@ -0,0 +1,643 @@
+---
+id: chat-completion
+title: Chat Completion
+hide_title: false
+hide_table_of_contents: false
+description: How to use the SAP Cloud SDK for AI to perform chat completion tasks using orchestration service from SAP AI Core.
+keywords:
+  - sap
+  - cloud
+  - sdk
+  - ai
+  - orchestration
+  - chat
+  - completion
+---
+
+The `@sap-ai-sdk/orchestration` package provides a client for the orchestration service of SAP AI Core.
+The orchestration service harmonizes the API of various generative AI models, enabling seamless integration and interaction with different models through a unified interface.
+Additionally, it provides features like [templating](#templating), [content filtering](#content-filtering), [grounding](#grounding) and many more to enhance the interaction with generative AI models.
+
+Find more details about orchestration workflow [here](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/orchestration-workflow).
+
+## Installation
+
+```
+$ npm install @sap-ai-sdk/orchestration
+```
+
+## Usage
+
+Initialize a client with proper orchestration configuration for each module.
+Below is an example for configuring the mandatory `llm` and `templating` modules.
+
+```ts
+import { OrchestrationClient } from '@sap-ai-sdk/orchestration';
+
+const orchestrationClient = new OrchestrationClient({
+  llm: {
+    model_name: 'gpt-4o'
+  },
+  templating: {
+    template: [
+      { role: 'user', content: 'Hello World! Why is this phrase so famous?' }
+    ]
+  }
+});
+```
+
+Here we specify the model name with a user message as prompt.
```

Comment:
```suggestion
Here, the model name is specified along with a user message as the prompt.
```

Its not a recommended practice to write instructional docs using first person. if really needed, use `you` (even though its often implied with imperative case)